src/loaders/common/loader_common_names.py

```diff
@@ -80,4 +80,7 @@
 OBJECTS_NAME_GENOME = "KBaseGenomes.Genome"
 
 # The metadata file name created during the Mash run
-MASH_METADATA = 'mash_run_metadata.json'
\ No newline at end of file
+MASH_METADATA = 'mash_run_metadata.json'
+
+# The fatal error file created if trait_counts.csv is not sucessfully generated
```

Comment:
```suggestion
# The fatal error file created if a data file cannot be not successfully processed
```

We should use this generally in other tools as well.

Reply:
Updated!

Reply:
:+1: